### DISCUSSION TITLE
关于cookies问题

### DISCUSSION THREAD

**Discussion by Brown05** [mikumifa/biliTickerBuy#409] [提问专区] [answered]
采用复制多个程序的方法进行多开，想要多个账号进行抢票。但是A程序的cookies调用不是在A的_internal（以往版本cookies在_internal），调用的是C:\Users\user\AppData\Local\Temp\gradio。导致无法多账号多开，有无解决方案

**Comment by mikumifa** [comment]
那个是老版本了，新版本直接在界面上改

**Reply by Brown05** [reply]
也试过在B上导入登录cookies，B登录成功后导致A报错，就是b的账号改动导致a的号也跟着换了。

**Reply by mikumifa** [reply]
复制整个压缩包？
不是单个exe

**Reply by Brown05** [reply]
对的，exe和_internal一起复制的

**Reply by Brown05** [reply]
所以说不知道是我的问题还是啥？

**Reply by Brown05** [reply, marked as answer]
似乎发现问题所在了，我的锅（捂脸.jpg），我是对已经启动过的exe和_internal的解包进行了复制，这样导致两个程序拉取了同一个路径。多开正确流程应该是在不同的文件夹对压缩包重新解压，程序在首次启动时会定义新路径从而实现多开。应该是这样理解吧？

**Reply by mikumifa** [reply]
差不多
